### PR TITLE
let -> const

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #16922 cos my suggestions didn't make it.

### CODE REVIEW

File: files/en-us/web/javascript/reference/errors/cant_access_lexical_declaration_before_init/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This happens within any block statement, when [`let`](/en-US/docs/Web/JavaScript
 
 ## Message
 
-```js
+```js plain
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/errors/cant_access_lexical_declaration_before_init/index.md:17]: Drive-by comment. I think this should be:
```suggestion
```plain
```

**Review by sideshowbarker — APPROVED** [review]
